Flutter/example_superapp/example/ios/Runner/FlutterPaymentViewController.swift

```diff
@@ -42,9 +42,29 @@ class FlutterPaymentViewController: FlutterViewController {
 			result(nil)
 			return
 		}
-		self.gxResultHandler = nil
+        if (call.method == "getInformation" ){
+            let args = call.arguments as? [String: Any]
+            guard let data = args?["data"] as? String else {
+                let errDesc = "Invalid result arguments: \(String(describing: call.arguments))"
```

Comment:
`result` is not called here. Also `errDesc` is not used.

Reply:
Fixed in https://github.com/genexus-books/gx-super-app/pull/63/commits/5eeec1208fa95fbf67a8bcd3c7808b20c2e66cb0